pandas/util/__init__.py

```diff
@@ -1,4 +1,60 @@
-from pandas.util._decorators import Appender, Substitution, cache_readonly  # noqa
+import importlib as _importlib
```

Comment:
umm why don't you actually just leave pandas.util.testing as a module itself? (and just re-import what you need). This is how we did all of the original deprecations.